inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/kafka/src/main/java/org/apache/inlong/sort/kafka/DynamicKafkaSerializationSchema.java

```diff
@@ -287,6 +297,26 @@ private void serializeWithDirtyHandle(Map<String, Object> baseMap, JsonNode root
         }
     }
 
+    public Set<SchemaChangeType> extractSchemaChangeType(JsonNode rootNode) {
+        Operation operation;
+        try {
+            JsonNode operationNode = Preconditions.checkNotNull(rootNode.get("operation"),
+                    "Operation node is null");
+            operation = Preconditions.checkNotNull(
+                    jsonDynamicSchemaFormat.objectMapper.convertValue(operationNode, new TypeReference<Operation>() {
+                    }), "Operation is null");
+        } catch (Exception e) {
+            LOG.warn("Extract Operation from origin data failed", e);
+            return null;
+        }
+        String originSchema = jsonDynamicSchemaFormat.extractDDL(rootNode);
+        Set<SchemaChangeType> types = SchemaChangeUtils.extractSchemaChangeTypes(operation);
+        if (types == null) {
```

Comment:
Condition 'types == null' is always 'false'

Reply:
Fixed